google-cloud-contrib/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/CloudStorageReadChannel.java

```diff
@@ -136,8 +141,13 @@ public int read(ByteBuffer dst) throws IOException {
   }
 
   private void sleepForAttempt(int attempt) {
+    long delay = 500;
+    // exponential backoff
```

Comment:
With maxChannelReopens == 3 and the exponential backoff as implemented here, will a starting delay of 500 typically provide enough backoff to recover from these errors, according to your testing?